jobs/kpi-forecasting/kpi-forecasting/kpi_forecasting.py

```diff
@@ -1,66 +1,30 @@
-"""
-Control script for KPI forecasting automation.
-Most of the time, this is not the script you should be editing
-Author: pmcmanis@mozilla.com
-Date: Mar 2022
-"""
+from inputs import CLI, YAML
+from models.prophet_forecast import ProphetForecast
+from metric_hub import MetricHub
 
-import argparse
 
-import yaml
-
-from Utils.ForecastDatasets import fetch_data
-from Utils.FitForecast import run_forecast
-from Utils.AutoArimaFit import run_forecast_arima
-from Utils.DBWriter import (
-    write_predictions_to_bigquery,
-    write_confidence_intervals_to_bigquery,
-)
-from Utils.AutoArimaDBWriter import write_arima_results_to_bigquery
-from Utils.PosteriorSampling import get_confidence_intervals
-
-
-def get_args() -> argparse.Namespace:
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        "-c", "--config", type=str, help="Path to configuration yaml file"
-    )
-    parser.add_argument(
-        "-r",
-        "--autoarima",
-        action="store_true",
-        help="Set this flag to use prophet instead of AutoArima",
-    )
-    return parser.parse_args()
+# A dictionary of available models in the `models` directory.
+MODELS = {
+    "prophet": ProphetForecast,
+}
 
 
 def main() -> None:
-    args = get_args()
-    with open(args.config, "r") as config_stream:
-        config = yaml.safe_load(config_stream)
+    # Load the config
+    config = YAML(filepath=CLI().args.config).data
```

Comment:
Is there a need to put YAML and CLI into separate modules? Both appear to be single functions and this appears like a quite a short module.

Reply:
That's a great question. I considered merging them, but I think that keeping them separate is helpful if the forecasting code is being run outside of a Docker context. For example, when developing locally in a Jupyter notebook, the code can instead look like:
```python
    config = YAML(filepath="local_config.yml").data
```

If it's okay, I'd prefer not to make this change for now.

Reply:
of course this is fine. The modularity is good, was just unsure if really unnecessary with so little code.